README.md

```diff
@@ -33,17 +35,17 @@ a commercial set of Instagram-like filters.
 
 ## How it works
 
-This method is based on
+This method provides an accurate color reconstruction, and is based on
 [three-dimensional lookup tables][wiki-luts] and their two-dimensional
 representation: [hald images][hald-image].
 The core idea is simple: a sample hald image with a uniform color distribution
-is processed using any target software with an unknown color
-transformation algorithm.
+is processed using target color filter with an unknown transformation algorithm.
 The processed hald image can then be used as a filter for a very accurate
 approximation of that target color transformation.
 
 A resulting hald image could then be used in various software such as
-GraphicsMagick or Adobe Photoshop and converted to the
+GraphicsMagick or Adobe Photoshop. You can use it in your iOS or macOS apps 
+with [CocoaLUT][CocoaLUT]. Hald images could be converted to the
```

Comment:
You can implement those hald images in your iOS or macOS app with CocoaLUT. Also, hald images